.github/workflows/halo.yaml

```diff
@@ -19,6 +19,10 @@ on:
     types:
       - published
 
```

Comment:
agreed, how about running the action manually by adding "workflow_dispatch:"

Reply:
That is definitely an option! 
I'm very interested as to why you would like to have this option enabled for this workflow?

Reply:
Well, actually, I misunderstood your modification, but it's not important.

How about add another condition to judge whether the push repo is the upstream or a fork, because if the fork repo enabled the GitHub action but it is still WIP, it may cause the action to fail and get the email notification.

Allowing running actions manually could avoid this.

Reply:
Hi @mashirot , welcome to propose another PR to resolve your problem.